flood_adapt/integrator/sfincs_adapter.py

```diff
@@ -105,9 +631,12 @@ def add_wind_forcing_from_grid(self, ds: xr.DataArray):
             - wind_v: northward wind velocity [m/s]
             - spatial_ref: CRS
         """
-        self.sf_model.setup_wind_forcing_from_grid(wind=ds)
+        self._model.setup_wind_forcing_from_grid(wind=ds)
 
-    def add_pressure_forcing_from_grid(self, ds: xr.DataArray):
+    def _add_pressure_forcing_from_grid(self, ds: xr.DataArray):
```

Comment:
Same as for wind above: This needs to be called for offshore model for historic_offshore events (i.e. water level forcing from model (oh, that makes it a little messier again)